api/app/tests/apps/api/v1/test_private_category_endpoint.py

```diff
@@ -58,6 +59,30 @@ def _assert_category_data(self, category, data):
             self.assertEqual(data['sla']['n_days'], slo.n_days)
             self.assertEqual(data['sla']['use_calendar_days'], slo.use_calendar_days)
 
+        self.assertIn('departments', data)
+        if category.departments.count() > 0:
```

Comment:
This code is never reached in the tests --- I ran with an assert False just after this `if` and all tests passed.

Reply:
Added an extra test to make sure this code is reached